docs/install.md

```diff
@@ -61,6 +63,14 @@ sure you have [configured `kubectl` to access your cluster](https://kubernetes.i
      underlying disk type. For more info, see the section below on "creating a storage class
      manually".
 
+1. Examine the sample `values.yaml` in the `deploy-sourcegraph` repo for additional starting
+   configuration values. Do not copy fields in if you don't need to change them; the values
+   in your user-provided file are merged in on top of this file. The existing resource
+   allocations are probably a reasonable starting point.
+
+   If you find you've made errors, you can update `values.yaml` using `helm upgrade`; for
```

Comment:
This is inaccurate. `helm upgrade` upgrades the running k8s cluster; it does not change `values.yaml`.

Reply:
Er. yes. "Deploy updated values.yaml to the cluster" or something.